app/css/cohortbuilder.css

```diff
@@ -229,6 +229,17 @@ a:hover {
     z-index: 5;
 }
 
-.no-border-radius {
-    border-radius: 0px;
+.search-history-title {
+    padding: 20px 0 10px 0;
+}
+
+.no-border {
+    border: 0px;
+}
+
+.btn-danger {
+    background: -webkit-linear-gradient(#ff6666, #ff3d3d);
+    background: -o-linear-gradient(#ff6666, #ff3d3d);
+    background: -moz-linear-gradient(#ff6666, #ff3d3d);
+    background: linear-gradient(#ff6666, #ff3d3d)
```

Comment:
I think we could add post-css to the webpack config instead and use the prefixer module instead of having to hard code all the browser prefixes

Reply:
Fixed!